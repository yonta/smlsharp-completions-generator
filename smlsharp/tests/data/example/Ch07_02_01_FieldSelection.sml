#X;

#X {X = 1.1, Y = 2.2};

fun f x = (#X x, #Y x);

