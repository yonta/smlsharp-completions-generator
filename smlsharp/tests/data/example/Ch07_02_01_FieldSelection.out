val it = fn : ['a#{X: 'b}, 'b. 'a -> 'b]

val it = 1.1 : real

val f = fn : ['a#{X: 'b, Y: 'c}, 'b, 'c. 'a -> 'b * 'c]
